client/src/utils.js

```diff
@@ -1 +1,8 @@
 export const formatPrice = (priceStr) => `$${priceStr.split('.')[0]}`;
+export const composeParams = (params) => {
+  let result = '?';
+  for (let param in params) {
+    result += `${param}=${typeof params[param] === 'string' ? '"' + params[param] + '"' : params[param]}&`
```

Comment:
This is an affront to all that is good in the world. Fix it or destroy everything.